src/PackageCompiler.jl

```diff
@@ -1064,7 +1061,7 @@ function bundle_artifacts(ctx, dest_dir, library_only; include_lazy_artifacts=tr
                         continue
                     end
                     meta = Pkg.Artifacts.artifact_meta(name, artifacts_toml_path)
-                    meta == nothing && continue
+                    meta === nothing && continue
```

Comment:
Do I remember correctly that `isnothing(meta)` would optimize just as well on 1.7, but not quite yet on 1.6? :)

Reply:
I thin it does, yes.